From 39e6b5605bd15db6a1f2f223dede9c8d2265140c Mon Sep 17 00:00:00 2001
From: Michael Orlitzky <michael@orlitzky.com>
Date: Mon, 9 Jan 2017 22:34:30 -0500
Subject: [PATCH 1/1] Use the default PHP error_reporting value.

The php_beautifier script triggers some strict/deprecation warnings
with newer versions of PHP. With error_reporting = E_ALL, they are
shown to the user. Instead, we can at least respect the default
value of error_reporting from php.ini which (on Gentoo, anyway)
hides strict/deprecation warnings.
---
 Beautifier.php         | 2 +-
 scripts/php_beautifier | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Beautifier.php b/Beautifier.php
index 584ac4d..90d8d0b 100755
--- a/Beautifier.php
+++ b/Beautifier.php
@@ -19,7 +19,7 @@
  * @license    http://www.php.net/license/3_0.txt  PHP License 3.0
  * @version    CVS: $Id:$
  */
-error_reporting(E_ALL);
+// error_reporting(E_ALL);
 // Before all, test the tokenizer extension
 if (!extension_loaded('tokenizer')) {
     throw new Exception("Compile php with tokenizer extension. Use --enable-tokenizer or don't use --disable-all on configure.");
diff --git a/scripts/php_beautifier b/scripts/php_beautifier
index 255a264..6e87f4e 100755
--- a/scripts/php_beautifier
+++ b/scripts/php_beautifier
@@ -52,7 +52,7 @@
     */
     require_once 'PHP/Beautifier/Batch.php';
     define('PHP_Beautifier_WINDOWS', substr(PHP_OS, 0, 3) == 'WIN');
-    error_reporting(E_ALL);
+    // error_reporting(E_ALL);
     // get log object
     $oLog = PHP_Beautifier_Common::getLog();
     
-- 
2.10.2

